examples_tests/35.GeometryCreator/main.cpp

```diff
@@ -221,17 +235,88 @@ int main()
 		std::make_pair(icosphereGeometry, gpuIcosphere)
 	};
 	
-	uint64_t lastFPSTime = 0;
-	while(device->run() && receiver.keepOpen())
+	QToQuitEventReceiver escaper;
```

Comment:
you actually dont need Q to quit anymore for Erfan's camera, just handle the window close mesage like the latest example 52 does on `input_channel_demo` branch

Reply:
Yes @AnastaZIuk
I think this way is better, but note that unlike input_channel_demo our "windowCallbackFunc" is in Common.h.

Also I think it would be good to have functions like "isMouseFocused" or "isKeyboardFocused" simply with storing bools and the help of functions like this : https://github.com/Devsh-Graphics-Programming/Nabla/blob/fdb4ec0ad935a6c78aa18e3a1d2bce3bb7e8c4a0/examples_tests/common/CommonAPI.h#L238

So we could choose if we want to process events when they're focused on the window or not.